web/src/custom_profile_fields_ui.ts

```diff
@@ -46,18 +47,44 @@ export function append_custom_profile_fields(element_id: string, user_id: number
             rendered_value: "",
         };
         const editable_by_user = current_user.is_admin || field.editable_by_user;
-        const is_select_field = field.type === all_field_types.SELECT.id;
+        const is_select_field =
+            field.type === all_field_types.SELECT.id ||
+            field.type === all_field_types.SELECT_MULTIPLE.id;
         const field_choices = [];
 
         if (is_select_field) {
-            const field_choice_dict = settings_components.select_field_data_schema.parse(
-                JSON.parse(field.field_data),
-            );
+            let field_choice_dict: Record<string, {text: string; order: string}> = {};
+            try {
+                field_choice_dict = settings_components.select_field_data_schema.parse(
+                    JSON.parse(field.field_data),
+                );
+            } catch {
+                field_choice_dict = {};
+            }
```

Comment:
I don't think we need this try-catch block. We expect server to validate `field_data` when creating and updating the field so we can expect `field.field_data` to always be a valid json.